WordPress/Classes/Extensions/NSString+Helpers.swift

```diff
@@ -19,4 +25,16 @@ extension NSString
         
         return uniqueSet
     }
+    
+    /// Validates the current string.
+    ///
+    /// -   returns: true if passes validation
+    ///
+    public func isValidEmail() -> Bool {
+        // From http://stackoverflow.com/a/3638271/1379066
+        let emailRegex = ".+@([A-Za-z0-9-]+\\.)+[A-Za-z]{2}[A-Za-z]*"
```

Comment:
RegEx updated (**HUGE** props to @aerych for helping me out there), and Unit Test with your samples added.

Thanks @koke!